server/core/votes-manager.js

```diff
@@ -29,6 +47,10 @@ module.exports = class VotesManager {
 	}
 
 	getVotesCount(song) {
+		if(this.partyMode) {
+			return parseInt(Math.random() * 100);

```

Comment:
😈  Good catch. There might've been a slight modification to the version that's currently running. For instance the now playing song might possibly receive the completely-random `80085` amount of votes. Nothing to worry about.